### PR TITLE
add courses section

### PR DESCRIPTION
**Pull request by 1Alisa1**

### CODE REVIEW

File: src/components/cardTariff/cardTariff.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import styles from './cardTariff.module.scss';
 
 export interface CardTariffProps {
-  title: string;
-  content: React.ReactNode;
+  title: React.ReactNode;
+  content: React.ReactNode | string;
```
Comment (N0rator) [inline src/components/cardTariff/cardTariff.tsx:5]: ``ReactNode`` type allows to pass string, so additional ``| string`` is not necessary.
Please, check if ``ReactNode`` type is enough nad, if yes, then remove ``| string``.

File: src/components/coursesSection/coursesSection.module.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        font-size: 48px;
+      }
+      @include small {
+        margin-bottom: 40px;
+        font-size: 32px;
+      }
+    }
+    .cardWrapper {
+      display: grid;
+      grid-template-columns: repeat(1, 1fr);
```
Comment (N0rator) [inline src/components/coursesSection/coursesSection.module.scss:39]: ``repeat`` usage here is not necessary and seems excess. Please, write just ``grid-template-columns: 1fr;``

**Review by N0rator — CHANGES_REQUESTED** [review]

File: src/components/coursesSection/coursesSection.module.scss (hunk 2 above)
Comment (1Alisa1) [reply src/components/coursesSection/coursesSection.module.scss:39]: fixed

File: src/components/cardTariff/cardTariff.tsx (hunk 1 above)
Comment (1Alisa1) [reply src/components/cardTariff/cardTariff.tsx:5]: fixed

**Pull request merged**